## Eusociality evolved at a much higher rate in haplodiploid insects

### Reference Information
This is the README file for the dataset associated with the manuscript "Eusociality evolved at a much higher rate in haplodiploid insects". This file uses Markdown syntax and has a .md extension

---
### General details of this README

* **File name:** README(.md)
* **Author:** Sachin Suresh
* **Contributors:** Timothy A. Linksvayer
* **Date created:** 01/01/2025 (mm/dd/yyyy)
* **Date modified:** 01/02/2025 (mm/dd/yyyy)
---
### Dataset Version and Release History

* **Version number:** 1.0.0
* **Date:** 01/01/2025 (mm/dd/yyyy)
---
### Dataset Attribution and Usage

* **Dataset Title:** Data for the manuscript 'Eusociality evolved at a much higher rate in haplodiploid insects'
* **Dataset collection and contribution:** Sachin Suresh
* **License:** CC0 1.0 Universal (CC0 1.0)
* **Dataset usage:** The authors sincerely request the researchers to reach out for re-using this dataset (https://creativecommons.org/publicdomain/zero/1.0/)
* **Dataset citation:**

    > Sachin Suresh, Timothy A. Linksvayer. 2025. Data for the article 'Eusociality evolved at a much higher rate in haplodiploid insects'
---
### Contact Information
  * Name: Sachin Suresh (Ph.D. student) (corresponding author)
  * Affiliations: Arizona State University; School of Life Sciences
  * ORCiD ID: https://orcid.org/0000-0002-5245-7008
  * Email: ssures53@asu.edu
  * Alternate Email: sachinddr3@gmail.com

#### Alternative Contact
  * Name: Dr. Timothy A. Linksvayer (corresponding author)
  * Affiliations: Arizona State University; School of Life Sciences
  * ORCiD ID: https://orcid.org/0000-0001-7034-1546
  * Email: tlinksva@asu.edu
  * Alternate Email: tlinksvayer@gmail.com
- - -
### Funding
This work received partial support from National Science Foundation (IOS 2128304) awarded to Timothy A. Linksvayer.
- - -

## Data and File Overview
### Summary Metrics
* **File count:** 52
* **Total file size:** 22.8 MB
* **File formats:** .csv, .tre, .R, .txt
 
### Table of Contents

| **Category**     | **Files**                                                                                                      |
|-------------------|----------------------------------------------------------------------------------------------------------------|
| **Trait Data Files** | eus_hap_data_20221130.csv  <br> halictidae_pruned_processed_pheno_df.csv <br> pheno_processed_df_20231023.csv <br> tip_labels_insect_tree_pruned_1000sp_eus_added_castor_20240130.csv <br> tree_of_sex_data_20221009.csv |
| **Tree Files**    | halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre <br> insect_phylogeny_2017_collapsed_calibrated.tre <br> insect_phylogeny_2017_collapsed_calibrated_trimmed_1000sp_eus_added_castor.tre |
| **R Scripts**     | **FitPagel**  <br> halictidae_pruned_hap_and_eus_depx_fitMk_ARD_fitzjohnpi_20231023.R <br> halictidae_pruned_hap_and_eus_depx_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R <br> halictidae_pruned_hap_and_eus_depx_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20241216.R <br> halictidae_pruned_hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_20231023.R <br> halictidae_pruned_hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R <br> halictidae_pruned_hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20241216.R <br> halictidae_pruned_hap_and_eus_depy_fitMk_ARD_fitzjohnpi_20231023.R <br> halictidae_pruned_hap_and_eus_depy_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R <br> halictidae_pruned_hap_and_eus_depy_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20241216.R <br> hap_and_eus_depx_fitMk_ARD_fitzjohnpi_20231023.R <br> hap_and_eus_depx_fitMk_ARD_fitzjohnpi_hapall_euskoos_20240323.R <br> hap_and_eus_depx_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R <br> hap_and_eus_depx_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20240129.R <br> hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_20231023.R <br> hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_hapall_euskoos_20240323.R <br> hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R <br> hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20240129.R <br> hap_and_eus_depy_fitMk_ARD_fitzjohnpi_20231023.R <br> hap_and_eus_depy_fitMk_ARD_fitzjohnpi_hapall_euskoos_20240323.R <br> hap_and_eus_depy_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R <br> hap_and_eus_depy_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20240129.R <br> results_consolidated_fulltree_hdall_eusall.R <br> results_consolidated_fulltree_hdall_euskoos.R <br> results_consolidated_fulltree_hdarrh_eusall.R <br> results_consolidated_fulltree_hdarrh_euskoos.R <br> results_consolidated_halictidae_pruned_hdall_eusall.R <br> results_consolidated_halictidae_pruned_hdarrh_eusall.R <br> results_consolidated_halictidae_pruned_hdarrh_euskoos.R <br> <br> **LTT** <br> lineages-through-time_plot_20240516.R <br> <br> **Simmap** <br> hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_fulltree_20231023.R <br> hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_hapall_halictidae_pruned_20240402.R <br> hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_fulltree_20240402.R <br> hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_halictidae_pruned.R <br> hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_fulltree_20231023.R <br> hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_halictidae_pruned_20231023.R <br> hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_hdall_euskoos_fulltree_20240402.R <br> results_simmap_fitMk_ARD_fitzjohnpi_fulltree.R <br> results_simmap_fitMk_ARD_fitzjohnpi_hapall_eusall_halictidae_pruned.R <br> results_simmap_fitMk_ARD_fitzjohnpi_hapall_euskoos_fulltree.R <br> results_simmap_fitMk_ARD_fitzjohnpi_haparrh_euskoos_fulltree.R <br> results_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_halictidae_pruned.R <br> results_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_fulltree.R <br> results_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_halictidae_pruned.R |
| **Tree calibration files**     | config_insect_calibration.txt
---
### Software
**Recommended software/tools:** RStudio 2023.06.0 or higher; R version 4.3.2 or higher
- - -

## File Details

### 1) eus_hap_data_20221130.csv
- **Description:** A comma-delimited file containing data on haplodiploid insects, including taxonomic details (e.g., Order, Family, Genus, Species) and attributes related to sociality and haplodiploidy.
- **File path:** /Data files/Trait data files/eus_hap_data_20221130.csv
- **Format(s):** `.csv`  
- **Size:** 192.8 KB  
- **Dimensions:** 1226 rows × 12 columns  
- **Variables:**
  - **Order:** The taxonomic order to which the species belongs.
  - **Family:** The taxonomic family to which the species belongs.
  - **Subfamily:** The taxonomic subfamily to which the species belongs.
  - **Tribe:** The taxonomic tribe, if applicable.
  - **Name:** Full taxonomic name of the species.
  - **Genus:** The genus of the species.
  - **Species:** The specific epithet of the species.
  - **Sociality:** Social organization type (e.g., solitary, eusocial).
  - **Source:Sociality:** The source of information for the sociality classification.
  - **Haplodiploidy:** Haplodiploidy status of the species.
  - **Source: Haplodiploidy:** The source of information for the haplodiploidy classification.


### 2) halictidae_pruned_processed_pheno_df.csv
- **Description:** A comma-delimited file containing data on sociality and ploidy of insects. This .csv file is specifically created to analyze evolutionary relationship between sociality and ploidy in an insect phylogeny, where the Halictidae family is pruned to "force" only two origins of eusociality. 
- **File path:** /Data files/Trait data files/halictidae_pruned_processed_pheno_df.csv
- **Format(s):** `.csv`  
- **Size:** 4421.5 KB  
- **Dimensions:** 49169 rows × 11 columns  
- **Variables:**
  - **name:** Full taxonomic name of the species.
  - **genus:** The genus to which the species belongs.
  - **species:** The specific epithet of the species.
  - **order:** The taxonomic order of the species.
  - **family:** The taxonomic family of the species.
  - **sociality:** The social organization type of the species (e.g., solitary, eusocial).
  - **haplodiploidy:** Ploidy type (e.g., arrhenotoky, paternal genome elimination, diploid).
  - **HD.arrhenotoky:** Stricter classification of haplodiploidy where only "arrhenotoky" is considered as haplodiploid ("HD") and rest are considered diploid ("DD")
  - **HD.all:** Broader classification of haplodiploidy where "arrhenotoky" and "paternal genome elimination" are considered as haplodiploid ("HD") and rest are considered diploid ("DD")
  - **Eusocial:** Classification of eusociality based on [Wilson (1971)](https://www.hup.harvard.edu/books/9780674454903)
  - **Eusocial.strict:** Strict classification of eusociality based on [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/full/10.1111/brv.12330)
  
  
### 3) pheno_processed_df_20231023.csv
- **Description:** A comma-delimited file containing data on sociality and ploidy of insects from the phylogeny used from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526)
- **File path:** /Data files/Trait data files/pheno_processed_df_20231023.csv
- **Format(s):** `.csv`  
- **Size:** 4875.1 KB  
- **Dimensions:** 49337 rows × 12 columns  
- **Variables:**
  - **name:** Full taxonomic name of the species.
  - **genus:** The genus to which the species belongs.
  - **species:** The specific epithet of the species.
  - **order:** The taxonomic order of the species.
  - **family:** The taxonomic family of the species.
  - **sociality:** The social organization type of the species (e.g., solitary, eusocial).
  - **haplodiploidy:** Ploidy type (e.g., arrhenotoky, paternal genome elimination, diploid).
  - **HD.arrhenotoky:** Stricter classification of haplodiploidy where only "arrhenotoky" is considered as haplodiploid ("HD") and rest are considered diploid ("DD").
  - **HD.all:** Broader classification of haplodiploidy where "arrhenotoky" and "paternal genome elimination" are considered as haplodiploid ("HD") and rest are considered diploid ("DD").
  - **Eusocial:** Classification of eusociality based on [Wilson (1971)](https://www.hup.harvard.edu/books/9780674454903).
  - **Eusocial.strict:** Strict classification of eusociality based on [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/full/10.1111/brv.12330).
  - **Eus.strict.koos.gawne:** Strict classification of eusociality based on [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/full/10.1111/brv.12330).
  
  
### 4) tip_labels_insect_tree_pruned_1000sp_eus_added_castor_20240130.csv
- **Description:** A comma-delimited file containing tip labels from a pruned insect phylogenetic tree, which is a representative subset of 1,000 species from the full 49,358 species-level phylogeny of [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526) with additional data on eusociality and haplodiploidy classifications. This dataset is used to generate Figure 1a in the manuscript
- **File path:** /Data files/Trait data files/tip_labels_insect_tree_pruned_1000sp_eus_added_castor_20240130.csv
- **Format(s):** `.csv`  
- **Size:** 31.2 KB  
- **Dimensions:** 1003 rows × 3 columns  
- **Variables:**
  - **label:** Tip labels corresponding to species or taxa in the phylogenetic tree.
  - **Eusocial:** Classification of eusociality based on [Wilson 1971](https://www.hup.harvard.edu/books/9780674454903).
  - **HD.all:** Broader classification of haplodiploidy where "arrhenotoky" and "paternal genome elimination" are considered as haplodiploid ("HD") and the rest are considered diploid ("DD").


### 5) tree_of_sex_data_20221009.csv
- **Description:** A comma-delimited file containing data from the [Tree of Sex database](https://www.treeofsex.org/data.html), including taxonomic classifications, sexual systems, karyotype information, and haplodiploidy attributes. 
- **File path:** /Data files/Trait data files/tree_of_sex_data_20221009.csv
- **Format(s):** `.csv`  
- **Size:** 9811.9 KB  
- **Dimensions:** 10972 rows × 26 columns  
- **Variables:**
  - **Order:** The taxonomic order to which the species belongs.
  - **Family:** The taxonomic family to which the species belongs.
  - **Genus:** The genus of the species.
  - **Species:** The specific epithet of the species.
  - **Sexual System:** Classification of the reproductive system (e.g., dioecy, hermaphroditism).
  - **source: Sexual System:** Source of information for the sexual system classification.
  - **Karyotype:** Karyotype description of the species.
  - **source: Karyotype:** Source of information for the karyotype data.
  - **notes: Karyotype:** Notes related to karyotype information.
  - **Genotypic:** Genotypic sex determination information.
  - **source: Genotypic:** Source of information for genotypic data.
  - **Haplodiploidy:** Classification of haplodiploidy status.
  - **notes: Haplodiploidy:** Notes related to haplodiploidy data.
  - **source: Haplodiploidy:** Source of information for haplodiploidy classification.
  - **Predicted ploidy:** Predicted ploidy level of the species.
  - **source: Predicted ploidy:** Source of information for predicted ploidy.
  - **Chromosome number (female) 2N:** Chromosome number in female individuals.
  - **source: Chromosome number (female) 2N:** Source of information for female chromosome numbers.
  - **notes: Chromosome number (female) 2N:** Notes related to female chromosome numbers.
  - **Chromosome number (male) 2N:** Chromosome number in male individuals.
  - **source: Chromosome number (male) 2N:** Source of information for male chromosome numbers.
  - **notes: Chromosome number (male) 2N:** Notes related to male chromosome numbers.
  - **notes, comments:** Additional notes or comments about the entry.
  - **entry name:** Contributor's name for the entry.
  - **entry email:** Contributor's email for the entry.
  - **citation:** Citation for the source of the data.


### 6) halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre 
- **Description:** A phylogenetic tree (.tre) file modified from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526) used for phylogenetic analysis in this manuscript. This file is used in analysis where the Halictidae family is pruned to "force" only two origins of eusociality.   
- **File path:** /Data files/Tree files/halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre
- **Format(s):** `.tre`  
- **Size:** 1913 KB 


### 7) insect_phylogeny_2017_collapsed_calibrated.tre
- **Description:** A phylogenetic tree (.tre) file modified from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526) used for phylogenetic analysis in this manuscript.   
- **File path:** /Data files/Tree files/insect_phylogeny_2017_collapsed_calibrated.tre
- **Format(s):** `.tre`  
- **Size:** 1931 KB 


### 8) insect_phylogeny_2017_collapsed_calibrated_trimmed_1000sp_eus_added_castor.tre
- **Description:** A phylogenetic tree (.tre) file modified from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526) used for phylogenetic analysis in this manuscript. This tree file is used to generate Figure 1a in the manuscript   
- **File path:** /Data files/Tree files/insect_phylogeny_2017_collapsed_calibrated_trimmed_1000sp_eus_added_castor.tre
- **Format(s):** `.tre`  
- **Size:** 43 KB 


### 9) halictidae_pruned_hap_and_eus_depx_fitMk_ARD_fitzjohnpi_20231023.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the phylogeny where the Halictidae family is pruned to "force" only two origins of eusociality.. This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depx** in the file name indicates that "sociality" is the dependent variable in the analysis.
- **File path:** /R scripts/FitPagel/halictidae_pruned_hap_and_eus_depx_fitMk_ARD_fitzjohnpi_20231023.R
- **Format(s):** `.R`  
- **Size:** 7.8 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** Phylogenetic tree file
  - **`halictidae_pruned_processed_pheno_df.csv`:** Processed phenotype data. 
- **Output Files:**
  - **`halictidae_pruned_dep_x_fitMk_ARD_fitzjohnpi_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)
  
  
### 10) halictidae_pruned_hap_and_eus_depx_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the phylogeny where the Halictidae family is pruned to "force" only two origins of eusociality.. This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depx** in the file name indicates that "sociality" is the dependent variable in the analysis. In this script, haplodiploidy is treated in the strict sense, using only **HD.arrhenotoky**.
- **File path:** /R scripts/FitPagel/halictidae_pruned_hap_and_eus_depx_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
- **Format(s):** `.R`  
- **Size:** 8.1 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** Phylogenetic tree file.
  - **`halictidae_pruned_processed_pheno_df.csv`:** Processed phenotype data. 
- **Output Files:**
  - **`halictidae_pruned_dep_x_fitMk_ARD_fitzjohnpi_haparrhenotoky_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)
  
  
### 11) halictidae_pruned_hap_and_eus_depx_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20241216.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the phylogeny where the Halictidae family is pruned to "force" only two origins of eusociality.. This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depx** in the file name indicates that "sociality" is the dependent variable in the analysis. In this script, haplodiploidy is treated in the strict sense, using **HD.arrhenotoky**, and eusociality is treated in the strict sense, using **Eusocial.strict**.
- **File path:** /R scripts/FitPagel/halictidae_pruned_hap_and_eus_depx_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20241216.R
- **Format(s):** `.R`  
- **Size:** 8.3 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** Phylogenetic tree file
  - **`halictidae_pruned_processed_pheno_df.csv`:** Processed phenotype data. 
- **Output Files:**
  - **`halictidae_pruned_dep_x_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)


### 12) halictidae_pruned_hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_20231023.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the phylogeny where the Halictidae family is pruned to "force" only two origins of eusociality.. This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depxy** in the file name indicates that both "sociality" and "ploidy" are interdependent variables in the analysis.
- **File path:** /R scripts/FitPagel/halictidae_pruned_hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_20231023.R
- **Format(s):** `.R`  
- **Size:** 8.4 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** Phylogenetic tree file
  - **`halictidae_pruned_processed_pheno_df.csv`:** Processed phenotype data. 
- **Output Files:**
  - **`halictidae_pruned_dep_xy_fitMk_ARD_fitzjohnpi_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform interdependent trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)


### 13) halictidae_pruned_hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the phylogeny where the Halictidae family is pruned to "force" only two origins of eusociality.. This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depxy** in the file name indicates that both "sociality" and "ploidy" are interdependent variables in the analysis. In this script, haplodiploidy is treated in the strict sense, using **HD.arrhenotoky**.
- **File path:** /R scripts/FitPagel/halictidae_pruned_hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
- **Format(s):** `.R`  
- **Size:** 8.2 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for Halictidae species.
  - **`halictidae_pruned_processed_pheno_df.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`halictidae_pruned_dep_xy_fitMk_ARD_fitzjohnpi_haparrhenotoky_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform interdependent trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)
  
  
### 14) halictidae_pruned_hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20241216.R
- **Description:**An R script designed to test the correlated evolution of Sociality and Ploidy across the phylogeny where the Halictidae family is pruned to "force" only two origins of eusociality.. This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depxy** in the file name indicates that both "sociality" and "ploidy" are interdependent variables in the analysis. In this script, haplodiploidy is treated in the strict sense, using **HD.arrhenotoky**, and eusociality is treated in the strict sense, using **Eusocial.strict**.
- **File path:** /R scripts/FitPagel/halictidae_pruned_hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20241216.R
- **Format(s):** `.R`  
- **Size:** 8.5 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for Halictidae species.
  - **`halictidae_pruned_processed_pheno_df.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`halictidae_pruned_dep_xy_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform interdependent trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)


### 15) halictidae_pruned_hap_and_eus_depy_fitMk_ARD_fitzjohnpi_20231023.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the phylogeny where the Halictidae family is pruned to "force" only two origins of eusociality.. This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depy** in the file name indicates that "ploidy" is the dependent variable in the analysis.
- **File path:** /R scripts/FitPagel/halictidae_pruned_hap_and_eus_depy_fitMk_ARD_fitzjohnpi_20231023.R
- **Format(s):** `.R`  
- **Size:** 8.0 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for Halictidae species.
  - **`halictidae_pruned_processed_pheno_df.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`halictidae_pruned_dep_y_fitMk_ARD_fitzjohnpi_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)


### 16) halictidae_pruned_hap_and_eus_depy_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the phylogeny where the Halictidae family is pruned to "force" only two origins of eusociality.. This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depy** in the file name indicates that "ploidy" is the dependent variable in the analysis. In this script, haplodiploidy is treated in the strict sense, using **HD.arrhenotoky**.
- **File path:** /R scripts/FitPagel/halictidae_pruned_hap_and_eus_depy_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
- **Format(s):** `.R`  
- **Size:** 8.1 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for Halictidae species.
  - **`halictidae_pruned_processed_pheno_df.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`halictidae_pruned_dep_y_fitMk_ARD_fitzjohnpi_haparrhenotoky_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)
  
  
### 17) halictidae_pruned_hap_and_eus_depy_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20241216.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the phylogeny where the Halictidae family is pruned to "force" only two origins of eusociality.. This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depy** in the file name indicates that "ploidy" is the dependent variable in the analysis. In this script, haplodiploidy is treated in the strict sense, using **HD.arrhenotoky**, and eusociality is treated in the strict sense, using **Eusocial.strict**.
- **File path:** /R scripts/FitPagel/halictidae_pruned_hap_and_eus_depy_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20241216.R
- **Format(s):** `.R`  
- **Size:** 8.4 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for Halictidae species.
  - **`halictidae_pruned_processed_pheno_df.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`halictidae_pruned_dep_y_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)
  
  
### 18) hap_and_eus_depx_fitMk_ARD_fitzjohnpi_20231023.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU.  The term **depx** in the file name indicates that "sociality" is the dependent variable in the analysis.
- **File path:** /R scripts/FitPagel/hap_and_eus_depx_fitMk_ARD_fitzjohnpi_20231023.R
- **Format(s):** `.R`  
- **Size:** 8.6 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`fulltree_dep_x_fitMk_ARD_fitzjohnpi_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)


### 19) hap_and_eus_depx_fitMk_ARD_fitzjohnpi_hapall_euskoos_20240323.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depx** in the file name indicates that "sociality" is the dependent variable in the analysis. In this script, haplodiploidy is treated in a broad sense, using **HD.all**, and eusociality is treated in the strict sense, following [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330).
- **File path:** /R scripts/FitPagel/hap_and_eus_depx_fitMk_ARD_fitzjohnpi_hapall_euskoos_20240323.R
- **Format(s):** `.R`  
- **Size:** 8.7 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`fulltree_dep_x_fitMk_ARD_fitzjohnpi_hdall_EusKoos_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)
  
  
### 20) hap_and_eus_depx_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depx** in the file name indicates that "sociality" is the dependent variable in the analysis. In this script, haplodiploidy is treated in the strict sense, using **HD.arrhenotoky**.
- **File path:** /R scripts/FitPagel/hap_and_eus_depx_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
- **Format(s):** `.R`  
- **Size:** 8.4 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`fulltree_dep_x_fitMk_ARD_fitzjohnpi_hdarrhenotoky_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)


### 21) hap_and_eus_depx_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20240129.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depx** in the file name indicates that "sociality" is the dependent variable in the analysis. In this script, haplodiploidy is treated in the strict sense, using **HD.arrhenotoky**, and eusociality is treated in the strict sense, following [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330).
- **File path:** /R scripts/FitPagel/hap_and_eus_depx_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20240129.R
- **Format(s):** `.R`  
- **Size:** 8.5 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`fulltree_dep_x_fitMk_ARD_fitzjohnpi_hdarrhenotoky_EusKoos_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)
  

### 22) hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_20231023.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depxy** in the file name indicates that both "sociality" and "ploidy" are interdependent variables in the analysis.
- **File path:** /R scripts/FitPagel/hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_20231023.R
- **Format(s):** `.R`  
- **Size:** 8.5 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`fulltree_dep_xy_fitMk_ARD_fitzjohnpi_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform interdependent trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)


### 23) hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_hapall_euskoos_20240323.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depxy** in the file name indicates that both "sociality" and "ploidy" are interdependent variables in the analysis. In this script, haplodiploidy is treated in a broad sense, using **HD.all**, and eusociality is treated in the strict sense, following [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330).
- **File path:** /R scripts/FitPagel/hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_hapall_euskoos_20240323.R
- **Format(s):** `.R`  
- **Size:** 8.9 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`fulltree_dep_xy_fitMk_ARD_fitzjohnpi_hdall_EusKoos_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform interdependent trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)


### 24) hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depxy** in the file name indicates that both "sociality" and "ploidy" are interdependent variables in the analysis. In this script, haplodiploidy is treated in the strict sense, using **HD.arrhenotoky**.
- **File path:** /R scripts/FitPagel/hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
- **Format(s):** `.R`  
- **Size:** 8.6 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`fulltree_dep_xy_fitMk_ARD_fitzjohnpi_hdarrhenotoky_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform interdependent trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)
  
### 25) hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20240129.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depxy** in the file name indicates that both "sociality" and "ploidy" are interdependent variables in the analysis. In this script, haplodiploidy is treated in the strict sense, using **HD.arrhenotoky**, and eusociality is treated in the strict sense, following [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330).
- **File path:** /R scripts/FitPagel/hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20240129.R
- **Format(s):** `.R`  
- **Size:** 8.6 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`fulltree_dep_xy_fitMk_ARD_fitzjohnpi_hdarrhenotoky_EusKoos_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform interdependent trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)


### 26) hap_and_eus_depy_fitMk_ARD_fitzjohnpi_20231023.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depy** in the file name indicates that "ploidy" is the dependent variable in the analysis.
- **File path:** /R scripts/FitPagel/hap_and_eus_depy_fitMk_ARD_fitzjohnpi_20231023.R
- **Format(s):** `.R`  
- **Size:** 8.3 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`fulltree_dep_y_fitMk_ARD_fitzjohnpi_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)


### 27) hap_and_eus_depy_fitMk_ARD_fitzjohnpi_hapall_euskoos_20240323.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depy** in the file name indicates that "ploidy" is the dependent variable in the analysis. In this script, haplodiploidy is treated in a broad sense, using **HD.all**, and eusociality is treated in the strict sense, following [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330).
- **File path:** /R scripts/FitPagel/hap_and_eus_depy_fitMk_ARD_fitzjohnpi_hapall_euskoos_20240323.R
- **Format(s):** `.R`  
- **Size:** 8.9 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`fulltree_dep_y_fitMk_ARD_fitzjohnpi_hdall_EusKoos_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)


### 28) hap_and_eus_depy_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depy** in the file name indicates that "ploidy" is the dependent variable in the analysis. In this script, haplodiploidy is treated in the strict sense, using **HD.arrhenotoky**.
- **File path:** /R scripts/FitPagel/hap_and_eus_depy_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
- **Format(s):** `.R`  
- **Size:** 8.3 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`fulltree_dep_y_fitMk_ARD_fitzjohnpi_hdarrhenotoky_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)


### 29) hap_and_eus_depy_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20240129.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depy** in the file name indicates that "ploidy" is the dependent variable in the analysis. In this script, haplodiploidy is treated in the strict sense, using **HD.arrhenotoky**, and eusociality is treated in the strict sense, following [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330).
- **File path:** /R scripts/FitPagel/hap_and_eus_depy_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20240129.R
- **Format(s):** `.R`  
- **Size:** 8.6 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`fulltree_dep_y_fitMk_ARD_fitzjohnpi_hdarrhenotoky_EusKoos_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)


### 30) results_consolidated_fulltree_hdall_eusall.R
- **Description:** An R script designed to consolidate and analyze the results of `fitPagel` runs across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). The script focuses on testing evolutionary correlations between Sociality and Ploidy using multiple dependency models (**depx**, **depy**, and **depxy**) and applies model comparison techniques to identify the best-fit model.
- **File path:** /R scripts/FitPagel/results_consolidated_fulltree_hdall_eusall.R
- **Format(s):** `.R`  
- **Size:** 10.2 KB  
- **Purpose:** 
  - To compare models of correlated evolution between eusociality and haplodiploidy.
  - Includes independent, dependent x, dependent y, and interdependent (x and y) models.
  - Performs statistical hypothesis testing and ANOVA to compare models.
  - Visualizes the best-supported model.
- **Input Files:**
  - **`fulltree_dep_xy_fitMk_ARD_fitzjohnpi_calib.RDS`:** Results of the interdependent model (x and y are interdependent).
  - **`fulltree_dep_x_fitMk_ARD_fitzjohnpi_calib.RDS`:** Results of the model where x is dependent on y.
  - **`fulltree_dep_y_fitMk_ARD_fitzjohnpi_calib.RDS`:** Results of the model where y is dependent on x.
- **Output Files:**
  - Model comparison statistics and visualization plots.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load required libraries and set up file paths for data and results.
  2. Load model result files for **depx**, **depy**, and **depxy** dependencies.
  3. Extract and summarize rate matrices, likelihoods, AIC scores, and hypothesis test results.
  4. Conduct ANOVA to compare models and identify the best-supported model.
  5. Visualize the best-supported model with rate-specific linewidths.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)


### 31) results_consolidated_fulltree_hdall_euskoos.R
- **Description:** An R script designed to consolidate and analyze the results of `fitPagel` runs across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). The script focuses on testing evolutionary correlations between Sociality and Ploidy using multiple dependency models (**depx**, **depy**, and **depxy**) and applies model comparison techniques to identify the best-fit model. In this analysis, haplodiploidy is treated in a broad sense (**HD.all**), and eusociality is treated in a strict sense (**Eus.strict**).
- **File path:** /R scripts/FitPagel/results_consolidated_fulltree_hdall_euskoos.R
- **Format(s):** `.R`  
- **Size:** 10.3 KB  
- **Purpose:** 
  - To compare models of correlated evolution between eusociality and haplodiploidy.
  - Includes independent, dependent x, dependent y, and interdependent (x and y) models.
  - Performs statistical hypothesis testing, ANOVA, and visualization of the best-supported model.
- **Input Files:**
  - **`fulltree_dep_xy_fitMk_ARD_fitzjohnpi_hdall_EusKoos_calib.RDS`:** Results of the interdependent model.
  - **`fulltree_dep_x_fitMk_ARD_fitzjohnpi_hdall_EusKoos_calib.RDS`:** Results of the model where x is dependent on y.
  - **`fulltree_dep_y_fitMk_ARD_fitzjohnpi_hdall_EusKoos_calib.RDS`:** Results of the model where y is dependent on x.
- **Output Files:**
  - Comparison statistics and plots of the best-supported model.
- **Key Libraries Used:**
  - `phytools`
- **Workflow:**
  1. Load necessary libraries and input files for the analysis.
  2. Load result files for **depx**, **depy**, and **depxy** dependency models.
  3. Extract and summarize rate matrices, likelihoods, AIC scores, and hypothesis test results.
  4. Conduct ANOVA to compare models and identify the best-supported model.
  5. Visualize the best-supported model with rate-specific linewidths.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)
  - [Boomsma and Gawne (2018) Eusociality](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330)


### 32) results_consolidated_fulltree_hdarrh_eusall.R
- **Description:** An R script designed to consolidate and analyze the results of `fitPagel` runs across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). The script focuses on testing evolutionary correlations between Sociality and Ploidy using multiple dependency models (**depx**, **depy**, and **depxy**) and applies model comparison techniques to identify the best-fit model. In this analysis, haplodiploidy is treated in the strict sense (**HD.arrhenotoky**), and eusociality is treated in a broad sense (**Eus.all**).
- **File path:** /R scripts/FitPagel/results_consolidated_fulltree_hdarrh_eusall.R
- **Format(s):** `.R`  
- **Size:** 10.1 KB  
- **Purpose:** 
  - To compare models of correlated evolution between eusociality and haplodiploidy.
  - Includes independent, dependent x, dependent y, and interdependent (x and y) models.
  - Performs statistical hypothesis testing, ANOVA, and visualization of the best-supported model.
- **Input Files:**
  - **`fulltree_dep_xy_fitMk_ARD_fitzjohnpi_hdarrhenotoky_calib.RDS`:** Results of the interdependent model.
  - **`fulltree_dep_x_fitMk_ARD_fitzjohnpi_hdarrhenotoky_calib.RDS`:** Results of the model where x is dependent on y.
  - **`fulltree_dep_y_fitMk_ARD_fitzjohnpi_hdarrhenotoky_calib.RDS`:** Results of the model where y is dependent on x.
- **Output Files:**
  - Comparison statistics and plots of the best-supported model.
- **Key Libraries Used:**
  - `phytools`
- **Workflow:**
  1. Load necessary libraries and input files for the analysis.
  2. Load result files for **depx**, **depy**, and **depxy** dependency models.
  3. Extract and summarize rate matrices, likelihoods, AIC scores, and hypothesis test results.
  4. Conduct ANOVA to compare models and identify the best-supported model.
  5. Visualize the best-supported model with rate-specific linewidths.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)


### 33) results_consolidated_fulltree_hdarrh_euskoos.R
- **Description:** An R script designed to consolidate and analyze the results of `fitPagel` runs across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). The script focuses on testing evolutionary correlations between Sociality and Ploidy using multiple dependency models (**depx**, **depy**, and **depxy**) and applies model comparison techniques to identify the best-fit model. In this analysis, haplodiploidy is treated in the strict sense (**HD.arrhenotoky**), and eusociality is treated in the strict sense (**EusKoos**) based on [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330).
- **File path:** /R scripts/FitPagel/results_consolidated_fulltree_hdarrh_euskoos.R
- **Format(s):** `.R`  
- **Size:** 10.5 KB  
- **Purpose:** 
  - To compare models of correlated evolution between eusociality and haplodiploidy.
  - Includes independent, dependent x, dependent y, and interdependent (x and y) models.
  - Performs statistical hypothesis testing, ANOVA, and visualization of the best-supported model.
- **Input Files:**
  - **`fulltree_dep_xy_fitMk_ARD_fitzjohnpi_hdarrhenotoky_euskoos_calib.RDS`:** Results of the interdependent model.
  - **`fulltree_dep_x_fitMk_ARD_fitzjohnpi_hdarrhenotoky_euskoos_calib.RDS`:** Results of the model where x is dependent on y.
  - **`fulltree_dep_y_fitMk_ARD_fitzjohnpi_hdarrhenotoky_euskoos_calib.RDS`:** Results of the model where y is dependent on x.
- **Output Files:**
  - Comparison statistics and plots of the best-supported model.
- **Key Libraries Used:**
  - `phytools`
- **Workflow:**
  1. Load necessary libraries and input files for the analysis.
  2. Load result files for **depx**, **depy**, and **depxy** dependency models.
  3. Extract and summarize rate matrices, likelihoods, AIC scores, and hypothesis test results.
  4. Conduct ANOVA to compare models and identify the best-supported model.
  5. Visualize the best-supported model with rate-specific linewidths.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)
  - [Boomsma and Gawne (2018) Eusociality](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330)


### 34) results_consolidated_halictidae_pruned_hdall_eusall.R
- **Description:** An R script designed to consolidate and analyze the results of `fitPagel` runs on phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526).to test the evolutionary correlation between Sociality and Ploidy. The Halictidae clade is pruned to enforce two origins of eusociality in this analysis. In this script, haplodiploidy is treated in a broad sense (**HD.all**), and eusociality is treated in a broad sense (**Eus.all**).
- **File path:** /R scripts/FitPagel/results_consolidated_halictidae_pruned_hdall_eusall.R
- **Format(s):** `.R`  
- **Size:** 10.5 KB  
- **Purpose:** 
  - To test evolutionary correlation between eusociality and haplodiploidy in Halictidae.
  - Implements multiple dependency models (**depx**, **depy**, and **depxy**) to identify the best-supported model.
  - Compares models using ANOVA and evaluates model likelihoods, AIC scores, and hypothesis test results.
  - Visualizes the best-supported model.
- **Input Files:**
  - **`halictidae_pruned_dep_xy_fitMk_ARD_fitzjohnpi_calib.RDS`:** Results of the interdependent model.
  - **`halictidae_pruned_dep_x_fitMk_ARD_fitzjohnpi_calib.RDS`:** Results of the model where x is dependent on y.
  - **`halictidae_pruned_dep_y_fitMk_ARD_fitzjohnpi_calib.RDS`:** Results of the model where y is dependent on x.
- **Output Files:**
  - Comparison statistics and visualizations of the best-supported model.
- **Key Libraries Used:**
  - `phytools`
- **Workflow:**
  1. Load necessary libraries and input files for the analysis.
  2. Load result files for **depx**, **depy**, and **depxy** dependency models.
  3. Extract and summarize rate matrices, likelihoods, AIC scores, and hypothesis test results.
  4. Conduct ANOVA to compare models and identify the best-supported model.
  5. Visualize the best-supported model with rate-specific linewidths.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)


### 35) results_consolidated_halictidae_pruned_hdarrh_eusall.R
- **Description:** An R script designed to consolidate and analyze the results of `fitPagel` runs on phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526).to test the evolutionary correlation between Sociality and Ploidy. The Halictidae clade is pruned to enforce two origins of eusociality in this analysis. In this script, haplodiploidy is treated in the strict sense (**HD.arrhenotoky**), and eusociality is treated in a broad sense (**Eus.all**).
- **File path:** /R scripts/FitPagel/results_consolidated_halictidae_pruned_hdarrh_eusall.R
- **Format(s):** `.R`  
- **Size:** 10.5 KB  
- **Purpose:** 
  - To test evolutionary correlation between eusociality and haplodiploidy in Halictidae.
  - Implements multiple dependency models (**depx**, **depy**, and **depxy**) to identify the best-supported model.
  - Compares models using ANOVA and evaluates model likelihoods, AIC scores, and hypothesis test results.
  - Visualizes the best-supported model.
- **Input Files:**
  - **`halictidae_pruned_dep_xy_fitMk_ARD_fitzjohnpi_calib.RDS`:** Results of the interdependent model.
  - **`halictidae_pruned_dep_x_fitMk_ARD_fitzjohnpi_calib.RDS`:** Results of the model where x is dependent on y.
  - **`halictidae_pruned_dep_y_fitMk_ARD_fitzjohnpi_calib.RDS`:** Results of the model where y is dependent on x.
- **Output Files:**
  - Comparison statistics and visualizations of the best-supported model.
- **Key Libraries Used:**
  - `phytools`
- **Workflow:**
  1. Load necessary libraries and input files for the analysis.
  2. Load result files for **depx**, **depy**, and **depxy** dependency models.
  3. Extract and summarize rate matrices, likelihoods, AIC scores, and hypothesis test results.
  4. Conduct ANOVA to compare models and identify the best-supported model.
  5. Visualize the best-supported model with rate-specific linewidths.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)


### 36) results_consolidated_halictidae_pruned_hdarrh_euskoos.R
- **Description:** An R script designed to consolidate and analyze the results of `fitPagel` runs on phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526).to test the evolutionary correlation between Sociality and Ploidy. The Halictidae clade is pruned to enforce two origins of eusociality in this analysis. In this script, haplodiploidy is treated in the strict sense (**HD.arrhenotoky**), and eusociality is treated in the strict sense (**EusKoos**) based on [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330).
- **File path:** /R scripts/FitPagel/results_consolidated_halictidae_pruned_hdarrh_euskoos.R
- **Format(s):** `.R`  
- **Size:** 10.8 KB  
- **Purpose:** 
  - To test evolutionary correlation between eusociality and haplodiploidy in Halictidae.
  - Implements multiple dependency models (**depx**, **depy**, and **depxy**) to identify the best-supported model.
  - Compares models using ANOVA and evaluates model likelihoods, AIC scores, and hypothesis test results.
  - Visualizes the best-supported model.
- **Input Files:**
  - **`halictidae_pruned_dep_xy_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_calib.RDS`:** Results of the interdependent model.
  - **`halictidae_pruned_dep_x_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_calib.RDS`:** Results of the model where x is dependent on y.
  - **`halictidae_pruned_dep_y_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_calib.RDS`:** Results of the model where y is dependent on x.
- **Output Files:**
  - Comparison statistics and visualizations of the best-supported model.
- **Key Libraries Used:**
  - `phytools`
- **Workflow:**
  1. Load necessary libraries and input files for the analysis.
  2. Load result files for **depx**, **depy**, and **depxy** dependency models.
  3. Extract and summarize rate matrices, likelihoods, AIC scores, and hypothesis test results.
  4. Conduct ANOVA to compare models and identify the best-supported model.
  5. Visualize the best-supported model with rate-specific linewidths.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)
  - [Boomsma and Gawne (2018) Eusociality](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330)


### 37) lineages-through-time_plot_20240516.R
- **Description:** An R script designed to create a lineage-through-time (LTT) plot for an insect phylogeny. The script focuses on visualizing phylogenetic dynamics over evolutionary time, using cumulative lineage counts and proportions of states to represent patterns of diversification and trait evolution.
- **File path:** /R scripts/LTT/lineages-through-time_plot_20240516.R
- **Format(s):** `.R`  
- **Size:** 12.6 KB  
- **Purpose:** 
  - To compute and visualize LTT for all states combined.
  - To generate state-specific LTT plots as cumulative area charts.
  - To plot proportions of lineages in each state over time.
- **Input Files:**
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_fulltree.RDS`:** Simmap object with state mappings.
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** Insect phylogeny file.
  - **`insect_ltt_updated.rds`:** Pre-computed LTT object.
- **Output Files:**
  - **`insect_ltt_updated.rds`:** Updated LTT object after computation.
  - LTT visualization plots stored in designated directories.
- **Key Libraries Used:**
  - `phytools`: For phylogenetic tree manipulation and LTT computation.
  - `tidyverse`: For data manipulation and custom plotting utilities.
- **Workflow:**
  1. Specify relevant directories for data, output, and figures.
  2. Load required data files (`RDS` and tree files).
  3. Compute LTT using `phytools` or load pre-computed LTT objects.
  4. Generate LTT plots, including:
     - Combined lineage counts.
     - State-specific cumulative area charts.
     - Proportional representation of states over time.
  5. Customize axes, grids, and visualization aesthetics.
  6. Save the computed LTT object and visualizations for downstream analysis.
- **References:**
  - [Phytools Documentation](https://cran.r-project.org/web/packages/phytools/phytools.pdf)


### 38) hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_fulltree_20231023.R
- **Description:** An R script designed to conduct stochastic character mapping (Simmap) analysis for Sociality and Ploidy across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). This script is optimized for streamlined execution on the SOL HPC system at ASU. In this script, haplodiploidy is treated in a broad sense (**HD.all**), and eusociality is treated in a broad sense (**Eusocial**).
- **File path:** /R scripts/Simmap/hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_fulltree_20231023.R
- **Format(s):** `.R`  
- **Size:** 12.3 KB  
- **Purpose:** 
  - To simulate evolutionary histories of traits using stochastic character mapping.
  - To save the resultant Simmap object for downstream analysis.
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset 
- **Output Files:**
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_fulltree.RDS`:** The resulting Simmap object containing 100 simulations.
- **Key Libraries Used:**
  - `phytools`
  - `dplyr`
- **Workflow:**
  1. Load the phylogenetic tree and phenotypic dataset.
  2. Prepare trait interaction data by combining haplodiploidy and eusociality traits.
  3. Conduct stochastic character mapping using `phytools::make.simmap` with the following settings:
     - Method: `fitMk`
     - Model: `ARD` (all-rates-different)
     - Pi: `fitzjohn`
     - Number of simulations: 100
  4. Save the resulting Simmap object for further downstream analysis.
- **References:**
  - [Phytools Documentation on Simmap](http://blog.phytools.org/2019/07/stochastic-character-mapping-with.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)


### 39) hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_hapall_halictidae_pruned_20240402.R
- **Description:** An R script designed to conduct stochastic character mapping (Simmap) analysis for Sociality and Ploidy on the phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). he Halictidae clade is pruned to enforce two origins of eusociality in this analysis. This script is optimized for streamlined execution on the SOL HPC system at ASU. In this analysis, haplodiploidy is treated in a broad sense (**HD.all**), and eusociality is treated in a broad sense (**Eusocial**).
- **File path:** /R scripts/Simmap/hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_hapall_halictidae_pruned_20240402.R
- **Format(s):** `.R`  
- **Size:** 12.4 KB  
- **Purpose:** 
  - To simulate evolutionary histories of traits using stochastic character mapping.
  - To save the resultant Simmap object for downstream analysis.
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** A pruned and calibrated phylogenetic tree file for Halictidae.
  - **`halictidae_pruned_processed_pheno_df.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_hapall_halictidae_pruned.RDS`:** The resulting Simmap object containing 100 simulations.
- **Key Libraries Used:**
  - `phytools`
  - `dplyr`
- **Workflow:**
  1. Load the pruned phylogenetic tree and phenotype dataset.
  2. Prepare trait interaction data by combining haplodiploidy and eusociality traits.
  3. Conduct stochastic character mapping using `phytools::make.simmap` with the following settings:
     - Method: `fitMk`
     - Model: `ARD` (all-rates-different)
     - Pi: `fitzjohn`
     - Number of simulations: 100
  4. Save the resulting Simmap object for further downstream analysis.
- **References:**
  - [Phytools Documentation on Simmap](http://blog.phytools.org/2019/07/stochastic-character-mapping-with.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)


### 40) hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_fulltree_20240402.R
- **Description:** An R script designed to conduct stochastic character mapping (Simmap) analysis for Sociality and Ploidy the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). This script is optimized for streamlined execution on the SOL HPC system at ASU. In this analysis, haplodiploidy is treated in the strict sense (**HD.arrhenotoky**), and eusociality is treated in the strict sense (**EusKoos**) based on [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330).
- **File path:** /R scripts/Simmap/hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_fulltree_20240402.R
- **Format(s):** `.R`  
- **Size:** 9.8 KB  
- **Purpose:** 
  - To simulate evolutionary histories of traits using stochastic character mapping.
  - To save the resultant Simmap object for downstream analysis.
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_fulltree.RDS`:** The resulting Simmap object containing 100 simulations.
- **Key Libraries Used:**
  - `phytools`
  - `dplyr`
- **Workflow:**
  1. Load the phylogenetic tree and phenotype dataset.
  2. Prepare trait interaction data by combining haplodiploidy and eusociality traits.
  3. Conduct stochastic character mapping using `phytools::make.simmap` with the following settings:
     - Method: `fitMk`
     - Model: `ARD` (all-rates-different)
     - Pi: `fitzjohn`
     - Number of simulations: 100
  4. Save the resulting Simmap object for further downstream analysis.
- **References:**
  - [Phytools Documentation on Simmap](http://blog.phytools.org/2019/07/stochastic-character-mapping-with.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)
  - [Boomsma and Gawne (2018) Eusociality](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330)


### 41) hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_halictidae_pruned.R
- **Description:** An R script designed to conduct stochastic character mapping (Simmap) analysis for Sociality and Ploidy on the phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). he Halictidae clade is pruned to enforce two origins of eusociality in this analysis. This script is optimized for streamlined execution on the SOL HPC system at ASU. In this analysis, haplodiploidy is treated in the strict sense (**HD.arrhenotoky**), and eusociality is treated in the strict sense (**Eusocial.strict**).
- **File path:** /R scripts/Simmap/hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_halictidae_pruned.R
- **Format(s):** `.R`  
- **Size:** 10.2 KB  
- **Purpose:** 
  - To simulate evolutionary histories of traits using stochastic character mapping.
  - To save the resultant Simmap object for downstream analysis.
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** A pruned and calibrated phylogenetic tree file for Halictidae.
  - **`halictidae_pruned_processed_pheno_df.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_halictidae_pruned.RDS`:** The resulting Simmap object containing 100 simulations.
- **Key Libraries Used:**
  - `phytools`
  - `dplyr`
- **Workflow:**
  1. Load the pruned phylogenetic tree and phenotype dataset.
  2. Prepare trait interaction data by combining haplodiploidy and eusociality traits.
  3. Conduct stochastic character mapping using `phytools::make.simmap` with the following settings:
     - Method: `fitMk`
     - Model: `ARD` (all-rates-different)
     - Pi: `fitzjohn`
     - Number of simulations: 100
  4. Save the resulting Simmap object for further downstream analysis.
- **References:**
  - [Phytools Documentation on Simmap](http://blog.phytools.org/2019/07/stochastic-character-mapping-with.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)
  - [Boomsma and Gawne (2018) Eusociality](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330)


### 42) hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_fulltree_20231023.R
- **Description:** An R script designed to conduct stochastic character mapping (Simmap) analysis for the traits Eusociality and Haplodiploidy across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). This script is optimized for streamlined execution on the SOL HPC system at ASU. In this analysis, haplodiploidy is treated in the strict sense (**HD.arrhenotoky**), and eusociality is treated in the broad sense (**Eusocial**).
- **File path:** /R scripts/Simmap/hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_fulltree_20231023.R
- **Format(s):** `.R`  
- **Size:** 8.9 KB  
- **Purpose:** 
  - To simulate evolutionary histories of traits using stochastic character mapping.
  - To save the resultant Simmap object for downstream analysis.
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_fulltree.RDS`:** The resulting Simmap object containing 100 simulations.
- **Key Libraries Used:**
  - `phytools`
  - `dplyr`
- **Workflow:**
  1. Load the phylogenetic tree and phenotype dataset.
  2. Prepare trait interaction data by combining haplodiploidy and eusociality traits.
  3. Conduct stochastic character mapping using `phytools::make.simmap` with the following settings:
     - Method: `fitMk`
     - Model: `ARD` (all-rates-different)
     - Pi: `fitzjohn`
     - Number of simulations: 100
  4. Save the resulting Simmap object for further downstream analysis.
- **References:**
  - [Phytools Documentation on Simmap](http://blog.phytools.org/2019/07/stochastic-character-mapping-with.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)


### 43) hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_halictidae_pruned_20231023.R
- **Description:** An R script designed to conduct stochastic character mapping (Simmap) analysis for Sociality and Ploidy on the phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). The Halictidae clade is pruned to enforce two origins of eusociality in this analysis. This script is optimized for streamlined execution on the SOL HPC system at ASU. In this analysis, haplodiploidy is treated in the strict sense (**HD.arrhenotoky**), and the phylogeny is pruned to enforce two independent origins of Eusociality.
- **File path:** /R scripts/Simmap/hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_halictidae_pruned_20231023.R
- **Format(s):** `.R`  
- **Size:** 9.2 KB  
- **Purpose:** 
  - To simulate evolutionary histories of traits using stochastic character mapping.
  - To save the resultant Simmap object for downstream analysis.
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** A pruned and calibrated phylogenetic tree file for Halictidae.
  - **`halictidae_pruned_processed_pheno_df.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_halictidae_pruned.RDS`:** The resulting Simmap object containing 100 simulations.
- **Key Libraries Used:**
  - `phytools`
  - `dplyr`
- **Workflow:**
  1. Load the pruned phylogenetic tree and phenotype dataset.
  2. Prepare trait interaction data by combining haplodiploidy and eusociality traits.
  3. Conduct stochastic character mapping using `phytools::make.simmap` with the following settings:
     - Method: `fitMk`
     - Model: `ARD` (all-rates-different)
     - Pi: `fitzjohn`
     - Number of simulations: 100
  4. Save the resulting Simmap object for further downstream analysis.
- **References:**
  - [Phytools Documentation on Simmap](http://blog.phytools.org/2019/07/stochastic-character-mapping-with.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)


### 44) hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_hdall_euskoos_fulltree_20240402.R
- **Description:** An R script designed to conduct stochastic character mapping (Simmap) analysis for the traits Eusociality and Haplodiploidy across the full phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). This script is optimized for streamlined execution on the SOL HPC system at ASU. In this analysis, haplodiploidy is treated in the broad sense (**HD.all**), and eusociality is treated in the strict sense (**Eus.strict**) based on [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330).
- **File path:** /R scripts/Simmap/hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_halictidae_pruned_20231023.R
- **Format(s):** `.R`  
- **Size:** 10.2 KB  
- **Purpose:** 
  - To simulate evolutionary histories of traits using stochastic character mapping.
  - To save the resultant Simmap object for downstream analysis.
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_hdall_euskoos_fulltree.RDS`:** The resulting Simmap object containing 100 simulations.
- **Key Libraries Used:**
  - `phytools`
  - `dplyr`
- **Workflow:**
  1. Load the phylogenetic tree and phenotype dataset.
  2. Prepare trait interaction data by combining haplodiploidy and eusociality traits.
  3. Conduct stochastic character mapping using `phytools::make.simmap` with the following settings:
     - Method: `fitMk`
     - Model: `ARD` (all-rates-different)
     - Pi: `fitzjohn`
     - Number of simulations: 100
  4. Save the resulting Simmap object for further downstream analysis.
- **References:**
  - [Phytools Documentation on Simmap](http://blog.phytools.org/2019/07/stochastic-character-mapping-with.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)
  - [Boomsma and Gawne (2018) Eusociality](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330)


### 45) results_simmap_fitMk_ARD_fitzjohnpi_fulltree.R
- **Description:** An R script designed to consolidate and visualize results from a Simmap analysis using the phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). Eusociality and Haplodiploidy, both defined in a broad sense (**HD.all** and **Eus.all**) for this analysis.
- **File path:** /R scripts/Simmap/results_simmap_fitMk_ARD_fitzjohnpi_fulltree.R
- **Format(s):** `.R`  
- **Size:** 15.2 KB  
- **Purpose:** 
  - To summarize lineage states (solitary/eusocial) across major insect clades.
  - To visualize origins of Eusociality using ridgeline plots.
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A calibrated phylogenetic tree for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_fulltree.RDS`:** Simmap object with stochastic character mapping results.
- **Output Files:**
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_fulltree_paraponera_removed.RDS`:** Updated Simmap object after removing problematic tips.
  - **`eusocial_origins_clades_hapall_eusall.svg`:** Ridgeline plot showing eusocial origins across clades.
- **Key Libraries Used:**
  - `phytools`
  - `ggplot2`
  - `RColorBrewer`
  - `viridis`
  - `patchwork`
  - `gridExtra`
  - `ggridges`
  - `svglite`
- **Workflow:**
  1. Load phylogenetic and phenotypic datasets.
  2. Identify major insect clades and classify them by Haplodiploidy (HD) or Diploidy (DD).
  3. Summarize Simmap results for each clade to count eusocial origins.
  4. Generate ridgeline plots to visualize eusociality across clades.
  5. Save the resulting visualizations and updated Simmap objects.
- **Visualization Details:**
  - X-axis: Number of eusocial origins.
  - Y-axis: Insect clades (e.g., ants, thrips, vespids).
  - Fill color: Represents ploidy type (haplodiploid or diploid).
- **References:**
  - [Phytools Simmap Documentation](http://blog.phytools.org/2019/07/stochastic-character-mapping-with.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)


### 46) results_simmap_fitMk_ARD_fitzjohnpi_hapall_eusall_halictidae_pruned.R
- **Description:** An R script designed to consolidate and visualize results from a Simmap analysis using the phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). The Halictidae clade is pruned to enforce two origins of eusociality in this analysis. Haplodiploidy is broadly defined (**HD.all**) and Eusociality is strictly defined (**EusKoos**) based on [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330).
- **File path:** /R scripts/Simmap/results_simmap_fitMk_ARD_fitzjohnpi_hapall_eusall_halictidae_pruned.R
- **Format(s):** `.R`  
- **Size:** 16.8 KB  
- **Purpose:** 
  - To summarize lineage states (solitary/eusocial) across major insect clades.
  - To visualize origins of Eusociality using ridgeline plots.
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A calibrated phylogenetic tree for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_hdall_euskoos_fulltree.RDS`:** Simmap object with stochastic character mapping results.
- **Output Files:**
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_hdall_euskoos_fulltree_paraponera_removed.RDS`:** Updated Simmap object after removing problematic tips.
  - **`eusocial_origins_clades_hapall_euskoos_fulltree.svg`:** Ridgeline plot showing eusocial origins across clades.
- **Key Libraries Used:**
  - `phytools`
  - `ggplot2`
  - `RColorBrewer`
  - `viridis`
  - `patchwork`
  - `gridExtra`
  - `ggridges`
  - `svglite`
- **Workflow:**
  1. Load phylogenetic and phenotypic datasets.
  2. Identify major insect clades and classify them by Haplodiploidy (HD) or Diploidy (DD).
  3. Summarize Simmap results for each clade to count eusocial origins.
  4. Generate ridgeline plots to visualize eusociality across clades.
  5. Save the resulting visualizations and updated Simmap objects.
- **Visualization Details:**
  - **X-axis:** Number of eusocial origins.
  - **Y-axis:** Insect clades (e.g., ants, thrips, vespids).
  - **Fill color:** Represents ploidy type (haplodiploid or diploid).
- **References:**
  - [Phytools Simmap Documentation](http://blog.phytools.org/2019/07/stochastic-character-mapping-with.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)
  - [Boomsma and Gawne (2018) Eusociality](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330)


### 47) results_simmap_fitMk_ARD_fitzjohnpi_hapall_euskoos_fulltree.R
- **Description:** An R script designed to consolidate and visualize results from a Simmap analysis using the phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). Haplodiploidy is broadly defined (**HD.all**) and Eusociality is strictly defined (**EusKoos**) based on [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330).
- **File path:** /R scripts/Simmap/results_simmap_fitMk_ARD_fitzjohnpi_hapall_euskoos_fulltree.R
- **Format(s):** `.R`  
- **Size:** 16.8 KB  
- **Purpose:** 
  - To summarize lineage states (solitary/eusocial) across major insect clades.
  - To visualize origins of Eusociality using ridgeline plots.
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A calibrated phylogenetic tree for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_hdall_euskoos_fulltree.RDS`:** Simmap object with stochastic character mapping results.
- **Output Files:**
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_hdall_euskoos_fulltree_paraponera_removed.RDS`:** Updated Simmap object after removing problematic tips.
  - **`eusocial_origins_clades_hapall_euskoos_fulltree.svg`:** Ridgeline plot showing eusocial origins across clades.
- **Key Libraries Used:**
  - `phytools`
  - `ggplot2`
  - `RColorBrewer`
  - `viridis`
  - `patchwork`
  - `gridExtra`
  - `ggridges`
  - `svglite`
- **Workflow:**
  1. Load phylogenetic and phenotypic datasets.
  2. Identify major insect clades and classify them by Haplodiploidy (HD) or Diploidy (DD).
  3. Summarize Simmap results for each clade to count eusocial origins.
  4. Generate ridgeline plots to visualize eusociality across clades.
  5. Save the resulting visualizations and updated Simmap objects.
- **Visualization Details:**
  - **X-axis:** Number of eusocial origins.
  - **Y-axis:** Insect clades (e.g., ants, thrips, vespids).
  - **Fill color:** Represents ploidy type (haplodiploid or diploid).
- **References:**
  - [Phytools Simmap Documentation](http://blog.phytools.org/2019/07/stochastic-character-mapping-with.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)
  - [Boomsma and Gawne (2018) Eusociality](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330)


### 48) results_simmap_fitMk_ARD_fitzjohnpi_haparrh_euskoos_fulltree.R
- **Description:** An R script designed to consolidate and visualize results from a Simmap analysis using the phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). Haplodiploidy is strictly defined (**HD.arrhenotoky**) and Eusociality is strictly defined (**EusKoos**) based on [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330).
- **File path:** /R scripts/Simmap/results_simmap_fitMk_ARD_fitzjohnpi_haparrh_euskoos_fulltree.R
- **Format(s):** `.R`  
- **Size:** 16.2 KB  
- **Purpose:** 
  - To summarize lineage states (solitary/eusocial) across major insect clades.
  - To visualize origins of Eusociality using ridgeline plots.
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A calibrated phylogenetic tree for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_fulltree.RDS`:** Simmap object with stochastic character mapping results.
- **Output Files:**
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_fulltree_paraponera_removed.RDS`:** Updated Simmap object after removing problematic tips.
  - **`eusocial_origins_clades_haparrh_euskoos_fulltree.svg`:** Ridgeline plot showing eusocial origins across clades.
- **Key Libraries Used:**
  - `phytools`
  - `ggplot2`
  - `RColorBrewer`
  - `viridis`
  - `patchwork`
  - `gridExtra`
  - `ggridges`
  - `svglite`
- **Workflow:**
  1. Load phylogenetic and phenotypic datasets.
  2. Identify major insect clades and classify them by Haplodiploidy (HD) or Diploidy (DD).
  3. Summarize Simmap results for each clade to count eusocial origins.
  4. Generate ridgeline plots to visualize eusociality across clades.
  5. Save the resulting visualizations and updated Simmap objects.
- **Visualization Details:**
  - **X-axis:** Number of eusocial origins.
  - **Y-axis:** Insect clades (e.g., ants, thrips, vespids).
  - **Fill color:** Represents ploidy type (haplodiploid or diploid).
- **References:**
  - [Phytools Simmap Documentation](http://blog.phytools.org/2019/07/stochastic-character-mapping-with.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)
  - [Boomsma and Gawne (2018) Eusociality](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330)


### 49) results_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_halictidae_pruned.R
- **Description:** An R script designed to consolidate and visualize results from a Simmap analysis using the phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). The Halictidae clade is pruned to enforce two origins of eusociality in this analysis. Haplodiploidy is strictly defined (**HD.arrhenotoky**) and Eusociality is strictly defined (**EusKoos**) based on [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330).
- **File path:** /R scripts/Simmap/results_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_halictidae_pruned.R
- **Format(s):** `.R`  
- **Size:** 16.8 KB  
- **Purpose:** 
  - To summarize lineage states (solitary/eusocial) across major insect clades.
  - To visualize origins of Eusociality using ridgeline plots.
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** A pruned and calibrated phylogenetic tree for Halictidae.
  - **`halictidae_pruned_processed_pheno_df.csv`:** A processed phenotype dataset containing species-level information.
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_halictidae_pruned.RDS`:** Simmap object with stochastic character mapping results.
- **Output Files:**
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_halictidae_pruned_paraponera_removed.RDS`:** Updated Simmap object after removing problematic tips.
  - **`eusocial_origins_clades_haparrh_euskoos_halpruned.svg`:** Ridgeline plot showing eusocial origins across clades.
- **Key Libraries Used:**
  - `phytools`
  - `ggplot2`
  - `RColorBrewer`
  - `viridis`
  - `patchwork`
  - `gridExtra`
  - `ggridges`
  - `svglite`
- **Workflow:**
  1. Load phylogenetic and phenotypic datasets.
  2. Identify major insect clades and classify them by Haplodiploidy (HD) or Diploidy (DD).
  3. Summarize Simmap results for each clade to count eusocial origins.
  4. Generate ridgeline plots to visualize eusociality across clades.
  5. Save the resulting visualizations and updated Simmap objects.
- **Visualization Details:**
  - **X-axis:** Number of eusocial origins.
  - **Y-axis:** Insect clades (e.g., ants, thrips, vespids).
  - **Fill color:** Represents ploidy type (haplodiploid or diploid).
- **References:**
  - [Phytools Simmap Documentation](http://blog.phytools.org/2019/07/stochastic-character-mapping-with.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)
  - [Boomsma and Gawne (2018) Eusociality](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330)


### 50) results_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_fulltree.R
- **Description:** An R script designed to consolidate and visualize results from a Simmap analysis using the phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). Haplodiploidy is strictly defined (**HD.arrhenotoky**) and Eusociality is broadly defined (**Eusocial**) based on [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330).
- **File path:** /R scripts/Simmap/results_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_fulltree.R
- **Format(s):** `.R`  
- **Size:** 16.5 KB  
- **Purpose:** 
  - To summarize lineage states (solitary/eusocial) across major insect clades.
  - To visualize origins of Eusociality using ridgeline plots.
- **Input Files:**
  - **`insect_phylogeny_2017_collapsed_calibrated.tre`:** A calibrated phylogenetic tree for insects.
  - **`pheno_processed_df_20231023.csv`:** A processed phenotype dataset containing species-level information.
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_fulltree_paraponera_removed.RDS`:** Updated Simmap object with "Paraponera clavata" removed.
- **Output Files:**
  - **`eusocial_origins_clades_haparrh_eusall_fulltree.svg`:** Ridgeline plot showing eusocial origins across clades.
- **Key Libraries Used:**
  - `phytools`
  - `ggplot2`
  - `RColorBrewer`
  - `viridis`
  - `patchwork`
  - `gridExtra`
  - `ggridges`
  - `svglite`
- **Workflow:**
  1. Load phylogenetic and phenotypic datasets.
  2. Identify major insect clades and classify them by Haplodiploidy (HD) or Diploidy (DD).
  3. Summarize Simmap results for each clade to count eusocial origins.
  4. Generate ridgeline plots to visualize eusociality across clades.
  5. Save the resulting visualizations and updated Simmap objects.
- **Visualization Details:**
  - **X-axis:** Number of eusocial origins.
  - **Y-axis:** Insect clades (e.g., ants, thrips, vespids).
  - **Fill color:** Represents ploidy type (haplodiploid or diploid).
- **References:**
  - [Phytools Simmap Documentation](http://blog.phytools.org/2019/07/stochastic-character-mapping-with.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)
  - [Boomsma and Gawne (2018) Eusociality](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330)


### 51) results_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_halictidae_pruned.R
- **Description:** An R script designed to consolidate and visualize results from a Simmap analysis using the phylogeny from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526). The Halictidae clade is pruned to enforce two origins of eusociality in this analysis. Haplodiploidy is strictly defined (**HD.arrhenotoky**) and Eusociality is broadly defined (**Eusocial**).
- **File path:** /R scripts/Simmap/results_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_halictidae_pruned.R
- **Format(s):** `.R`  
- **Size:** 16.5 KB  
- **Purpose:** 
  - To summarize lineage states (solitary/eusocial) across major insect clades.
  - To visualize origins of Eusociality using ridgeline plots..
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** A pruned and calibrated phylogenetic tree for Halictidae.
  - **`halictidae_pruned_processed_pheno_df.csv`:** A processed phenotype dataset containing species-level information.
  - **`hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_halictidae_pruned_paraponera_removed.RDS`:** Simmap object with stochastic character mapping results.
- **Output Files:**
  - **`eusocial_origins_clades_haparrh_eusall_halpruned.svg`:** Ridgeline plot showing eusocial origins across clades.
- **Key Libraries Used:**
  - `phytools`
  - `ggplot2`
  - `RColorBrewer`
  - `viridis`
  - `patchwork`
  - `gridExtra`
  - `ggridges`
  - `svglite`
- **Workflow:**
  1. Load phylogenetic and phenotypic datasets.
  2. Identify major insect clades and classify them by Haplodiploidy (HD) or Diploidy (DD).
  3. Summarize Simmap results for each clade to count eusocial origins.
  4. Generate ridgeline plots to visualize eusociality across clades.
  5. Save the resulting visualizations and updated Simmap objects.
- **Visualization Details:**
  - **X-axis:** Number of eusocial origins.
  - **Y-axis:** Insect clades (e.g., ants, thrips, vespids).
  - **Fill color:** Represents ploidy type (haplodiploid or diploid).
- **References:**
  - [Phytools Simmap Documentation](http://blog.phytools.org/2019/07/stochastic-character-mapping-with.html)
  - [Chesters (2017) Phylogeny Source](https://academic.oup.com/sysbio/article/66/3/426/2684526)
  - [Boomsma and Gawne (2018) Eusociality](https://onlinelibrary.wiley.com/doi/10.1111/brv.12330)


### 52) config_insect_calibration.txt
- **Description:** A configuration file for running **treePL**, a software designed for phylogenetic dating using penalized likelihood methods. This file specifies calibration constraints, input tree details, computational parameters, and optimization settings. The file is associated with insect phylogeny calibration for divergence time estimation.
- **File path:** /Tree calibration files/config_insect_calibration.txt
- **Format(s):** `.txt`
- **Size:** 3.2 KB
- **Purpose:**
  - To define minimum and maximum age constraints for specific nodes in the phylogenetic tree.
  - To configure computational resources, such as the number of threads (`nthreads = 10`).
  - To enable detailed output for optimization and cross-validation analyses.
- **Key Sections:**
  1. **Input ML Tree:** Specifies the maximum likelihood (ML) tree file (`treefile`).
  2. **General Commands:**
     - `thorough`: Ensures an exhaustive search for optimization.
     - `log_pen`: Enables logging of penalties during the optimization process.
  3. **Calibrations:**
     - 18 calibration constraints are defined, each with minimum (`min`) and maximum (`max`) age bounds for specific most recent common ancestor (MRCA) nodes.
     - Example: 
       - **Calibration_1:** Ceylonosticta_austeni ↔ Neopetalia_punctata  
         - Min: 228 Ma, Max: 479 Ma.
  4. **Priming Command:** Enables priming mode for smoother optimization (`prime`).
  5. **Optimization Parameters:**
     - Comments suggest settings for optimization (`opt`, `optad`, `optcvad`) and smoothing (`smooth`), though these are currently not active.
  6. **Cross-Validation Analysis:**
     - Includes placeholders for cross-validation iterations and parameters, but these settings are commented out.
  7. **Output File** 

- **References:**
  - [TreePL GitHub Documentation](https://github.com/blackrim/treePL/wiki)
  - [TreePL Paper on arXiv](https://arxiv.org/ftp/arxiv/papers/2008/2008.07054.pdf)
