## Eusociality evolved at a much higher rate in haplodiploid insects

### Reference Information
This is the README file for the dataset associated with the manuscript "Eusociality evolved at a much higher rate in haplodiploid insects". This file uses Markdown syntax and has a .md extension

---
### General details of this README

* **File name:** README(.md)
* **Author:** Sachin Suresh
* **Contributors:** Timothy A. Linksvayer
* **Date created:** 01/01/2025 (mm/dd/yyyy)
* **Date modified:** 01/01/2025 (mm/dd/yyyy)
---
### Dataset Version and Release History

* **Version number:** 1.0.0
* **Date:** 01/01/2025 (mm/dd/yyyy)
---
### Dataset Attribution and Usage

* **Dataset Title:** Data for the manuscript 'Eusociality evolved at a much higher rate in haplodiploid insects'
* **Dataset collection and contribution:** Sachin Suresh
* **License:** CC0 1.0 Universal (CC0 1.0)
* **Dataset usage:** The authors sincerely request the researchers to reach out for re-using this dataset (https://creativecommons.org/publicdomain/zero/1.0/)
* **Dataset citation:**

    > Sachin Suresh, Timothy A. Linksvayer. 2023. Data for the article 'Eusociality evolved at a much higher rate in haplodiploid insects'
---
### Contact Information
  * Name: Sachin Suresh (Ph.D. student) (corresponding author)
  * Affiliations: Arizona State University; School of Life Sciences
  * ORCiD ID: https://orcid.org/0000-0002-5245-7008
  * Email: ssures53@asu.edu
  * Alternate Email: sachinddr3@gmail.com

#### Alternative Contact
  * Name: Dr. Timothy A. Linksvayer (corresponding author)
  * Affiliations: Arizona State University; School of Life Sciences
  * ORCiD ID: https://orcid.org/0000-0001-7034-1546
  * Email: tlinksva@asu.edu
  * Alternate Email: tlinksvayer@gmail.com
- - -
### Funding
This work received partial support from National Science Foundation (IOS 2128304) awarded to Timothy A. Linksvayer.
- - -

## Data and File Overview
### Summary Metrics
* **File count:** 50
* **Total file size:** 22.6 MB
* **File formats:** .csv, .tre, .R
 
### Table of Contents

* Trait data files
  * eus_hap_data_20221130.csv
  * halictidae_pruned_processed_pheno_df.csv
  * pheno_processed_df_20231023.csv
  * tip_labels_insect_tree_pruned_1000sp_eus_added_castor_20240130.csv
  * tree_of_sex_data_20221009.csv

* Tree file
  * halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre
  * insect_phylogeny_2017_collapsed_calibrated.tre
  * insect_phylogeny_2017_collapsed_calibrated_trimmed_1000sp_eus_added_castor.tre
  
* R scripts
  * FitPagel
    * halictidae_pruned_hap_and_eus_depx_fitMk_ARD_fitzjohnpi_20231023.R
    * halictidae_pruned_hap_and_eus_depx_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
    * halictidae_pruned_hap_and_eus_depx_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20241216.R
    * halictidae_pruned_hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_20231023.R
    * halictidae_pruned_hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
    * halictidae_pruned_hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20241216.R
    * halictidae_pruned_hap_and_eus_depy_fitMk_ARD_fitzjohnpi_20231023.R
    * halictidae_pruned_hap_and_eus_depy_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
    * halictidae_pruned_hap_and_eus_depy_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20241216.R
    * hap_and_eus_depx_fitMk_ARD_fitzjohnpi_20231023.R
    * hap_and_eus_depx_fitMk_ARD_fitzjohnpi_hapall_euskoos_20240323.R
    * hap_and_eus_depx_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
    * hap_and_eus_depx_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20240129.R
    * hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_20231023.R
    * hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_hapall_euskoos_20240323.R
    * hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
    * hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20240129.R
    * hap_and_eus_depy_fitMk_ARD_fitzjohnpi_20231023.R
    * hap_and_eus_depy_fitMk_ARD_fitzjohnpi_hapall_euskoos_20240323.R
    * hap_and_eus_depy_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
    * hap_and_eus_depy_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20240129.R
    * results_consolidated_fulltree_hdall_eusall.R
    * results_consolidated_fulltree_hdall_euskoos.R
    * results_consolidated_fulltree_hdarrh_eusall.R
    * results_consolidated_fulltree_hdarrh_euskoos.R
    * results_consolidated_halictidae_pruned_hdall_eusall.R
    * results_consolidated_halictidae_pruned_hdarrh_eusall.R
    * results_consolidated_halictidae_pruned_hdarrh_euskoos.R
    
  * LTT
    * lineages-through-time_plot_20240516.R
    
  * Simmap
    * hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_fulltree_20231023.R
    * hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_hapall_halictidae_pruned_20240402.R
    * hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_fulltree_20240402.R
    * hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_halictidae_pruned.R
    * hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_fulltree_20231023.R
    * hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_halictidae_pruned_20231023.R
    * hap_and_eus_simmap_fitMk_ARD_fitzjohnpi_hdall_euskoos_fulltree_20240402.R
    * results_simmap_fitMk_ARD_fitzjohnpi_fulltree.R
    * results_simmap_fitMk_ARD_fitzjohnpi_hapall_eusall_halictidae_pruned.R
    * results_simmap_fitMk_ARD_fitzjohnpi_hapall_euskoos_fulltree.R
    * results_simmap_fitMk_ARD_fitzjohnpi_haparrh_euskoos_fulltree.R
    * results_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_fulltree.R
    * results_simmap_fitMk_ARD_fitzjohnpi_haparrhenotoky_halictidae_pruned.R

---
### Software
**Recommended software/tools:** RStudio 2023.06.0 or higher; R version 4.3.2 or higher
- - -

## File Details

### 1) eus_hap_data_20221130.csv
- **Description:** A comma-delimited file containing data on haplodiploid insects, including taxonomic details (e.g., Order, Family, Genus, Species) and attributes related to sociality and haplodiploidy.
- **File path:** /Data files/Trait data files/eus_hap_data_20221130.csv
- **Format(s):** `.csv`  
- **Size:** 192.8 KB  
- **Dimensions:** 1226 rows × 12 columns  
- **Variables:**
  - **Order:** The taxonomic order to which the species belongs.
  - **Family:** The taxonomic family to which the species belongs.
  - **Subfamily:** The taxonomic subfamily to which the species belongs.
  - **Tribe:** The taxonomic tribe, if applicable.
  - **Name:** Full taxonomic name of the species.
  - **Genus:** The genus of the species.
  - **Species:** The specific epithet of the species.
  - **Sociality:** Social organization type (e.g., solitary, eusocial).
  - **Source:Sociality:** The source of information for the sociality classification.
  - **Haplodiploidy:** Haplodiploidy status of the species.
  - **Source: Haplodiploidy:** The source of information for the haplodiploidy classification.


### 2) halictidae_pruned_processed_pheno_df.csv
- **Description:** A comma-delimited file containing data on sociality and ploidy of insects. This .csv file is specifically created to analyze evolutionary relationship between sociality and ploidy in an insect phylogeny, where the Halictidae family is pruned to "force" only two origins of eusociality. 
- **File path:** /Data files/Trait data files/halictidae_pruned_processed_pheno_df.csv
- **Format(s):** `.csv`  
- **Size:** 4421.5 KB  
- **Dimensions:** 49169 rows × 11 columns  
- **Variables:**
  - **name:** Full taxonomic name of the species.
  - **genus:** The genus to which the species belongs.
  - **species:** The specific epithet of the species.
  - **order:** The taxonomic order of the species.
  - **family:** The taxonomic family of the species.
  - **sociality:** The social organization type of the species (e.g., solitary, eusocial).
  - **haplodiploidy:** Ploidy type (e.g., arrhenotoky, paternal genome elimination, diploid).
  - **HD.arrhenotoky:** Stricter classification of haplodiploidy where only "arrhenotoky" is considered as haplodiploid ("HD") and rest are considered diploid ("DD")
  - **HD.all:** Broader classification of haplodiploidy where "arrhenotoky" and "paternal genome elimination" are considered as haplodiploid ("HD") and rest are considered diploid ("DD")
  - **Eusocial:** Classification of eusociality based on [Wilson (1971)](https://www.hup.harvard.edu/books/9780674454903)
  - **Eusocial.strict:** Strict classification of eusociality based on [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/full/10.1111/brv.12330)
  
  
### 3) pheno_processed_df_20231023.csv
- **Description:** A comma-delimited file containing processed phenotypic and taxonomic data for various species, focusing on classifications such as sociality and haplodiploidy in insects
- **File path:** /Data files/Trait data files/pheno_processed_df_20231023.csv
- **Format(s):** `.csv`  
- **Size:** 4875.1 KB  
- **Dimensions:** 49337 rows × 12 columns  
- **Variables:**
  - **name:** Full taxonomic name of the species.
  - **genus:** The genus to which the species belongs.
  - **species:** The specific epithet of the species.
  - **order:** The taxonomic order of the species.
  - **family:** The taxonomic family of the species.
  - **sociality:** The social organization type of the species (e.g., solitary, eusocial).
  - **haplodiploidy:** Ploidy type (e.g., arrhenotoky, paternal genome elimination, diploid).
  - **HD.arrhenotoky:** Stricter classification of haplodiploidy where only "arrhenotoky" is considered as haplodiploid ("HD") and rest are considered diploid ("DD").
  - **HD.all:** Broader classification of haplodiploidy where "arrhenotoky" and "paternal genome elimination" are considered as haplodiploid ("HD") and rest are considered diploid ("DD").
  - **Eusocial:** Classification of eusociality based on [Wilson (1971)](https://www.hup.harvard.edu/books/9780674454903).
  - **Eusocial.strict:** Strict classification of eusociality based on [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/full/10.1111/brv.12330).
  - **Eus.strict.koos.gawne:** Strict classification of eusociality based on [Boomsma and Gawne (2018)](https://onlinelibrary.wiley.com/doi/full/10.1111/brv.12330).
  
  
### 4) tip_labels_insect_tree_pruned_1000sp_eus_added_castor_20240130.csv
- **Description:** A comma-delimited file containing tip labels from a pruned insect phylogenetic tree, which is a representative subset of 1,000 species from the full 49,358 species-level phylogeny of [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526) with additional data on eusociality and haplodiploidy classifications. This dataset is used to generate Figure 1a in the manuscript
- **File path:** /Data files/Trait data files/tip_labels_insect_tree_pruned_1000sp_eus_added_castor_20240130.csv
- **Format(s):** `.csv`  
- **Size:** 31.2 KB  
- **Dimensions:** 1003 rows × 3 columns  
- **Variables:**
  - **label:** Tip labels corresponding to species or taxa in the phylogenetic tree.
  - **Eusocial:** Classification of eusociality based on [Wilson 1971](https://www.hup.harvard.edu/books/9780674454903).
  - **HD.all:** Broader classification of haplodiploidy where "arrhenotoky" and "paternal genome elimination" are considered as haplodiploid ("HD") and the rest are considered diploid ("DD").


### 5) tree_of_sex_data_20221009.csv
- **Description:** A comma-delimited file containing data from the [Tree of Sex database](https://www.treeofsex.org/data.html), including taxonomic classifications, sexual systems, karyotype information, and haplodiploidy attributes. 
- **File path:** /Data files/Trait data files/tree_of_sex_data_20221009.csv
- **Format(s):** `.csv`  
- **Size:** 9811.9 KB  
- **Dimensions:** 10972 rows × 26 columns  
- **Variables:**
  - **Order:** The taxonomic order to which the species belongs.
  - **Family:** The taxonomic family to which the species belongs.
  - **Genus:** The genus of the species.
  - **Species:** The specific epithet of the species.
  - **Sexual System:** Classification of the reproductive system (e.g., dioecy, hermaphroditism).
  - **source: Sexual System:** Source of information for the sexual system classification.
  - **Karyotype:** Karyotype description of the species.
  - **source: Karyotype:** Source of information for the karyotype data.
  - **notes: Karyotype:** Notes related to karyotype information.
  - **Genotypic:** Genotypic sex determination information.
  - **source: Genotypic:** Source of information for genotypic data.
  - **Haplodiploidy:** Classification of haplodiploidy status.
  - **notes: Haplodiploidy:** Notes related to haplodiploidy data.
  - **source: Haplodiploidy:** Source of information for haplodiploidy classification.
  - **Predicted ploidy:** Predicted ploidy level of the species.
  - **source: Predicted ploidy:** Source of information for predicted ploidy.
  - **Chromosome number (female) 2N:** Chromosome number in female individuals.
  - **source: Chromosome number (female) 2N:** Source of information for female chromosome numbers.
  - **notes: Chromosome number (female) 2N:** Notes related to female chromosome numbers.
  - **Chromosome number (male) 2N:** Chromosome number in male individuals.
  - **source: Chromosome number (male) 2N:** Source of information for male chromosome numbers.
  - **notes: Chromosome number (male) 2N:** Notes related to male chromosome numbers.
  - **notes, comments:** Additional notes or comments about the entry.
  - **entry name:** Contributor's name for the entry.
  - **entry email:** Contributor's email for the entry.
  - **citation:** Citation for the source of the data.


### 6) halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre 
- **Description:** A phylogenetic tree (.tre) file modified from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526) used for phylogenetic analysis in this manuscript. This file is used in analysis where the Halictidae family is pruned to "force" only two origins of eusociality.   
- **File path:** /Data files/Tree files/halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre
- **Format(s):** `.tre`  
- **Size:** 1913 KB 


### 7) insect_phylogeny_2017_collapsed_calibrated.tre
- **Description:** A phylogenetic tree (.tre) file modified from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526) used for phylogenetic analysis in this manuscript.   
- **File path:** /Data files/Tree files/insect_phylogeny_2017_collapsed_calibrated.tre
- **Format(s):** `.tre`  
- **Size:** 1931 KB 


### 8) insect_phylogeny_2017_collapsed_calibrated_trimmed_1000sp_eus_added_castor.tre
- **Description:** A phylogenetic tree (.tre) file modified from [Chesters (2017)](https://academic.oup.com/sysbio/article/66/3/426/2684526) used for phylogenetic analysis in this manuscript. This tree file is used to generate Figure 1a in the manuscript   
- **File path:** /Data files/Tree files/insect_phylogeny_2017_collapsed_calibrated_trimmed_1000sp_eus_added_castor.tre
- **Format(s):** `.tre`  
- **Size:** 43 KB 


### 9) halictidae_pruned_hap_and_eus_depx_fitMk_ARD_fitzjohnpi_20231023.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the phylogeny where the Halictidae family is pruned to "force" only two origins of eusociality.. This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depx** in the file name indicates that "sociality" is the dependent variable in the analysis.
- **Format(s):** `.R`  
- **Size:** 7.8 KB  
- **Purpose:** 
  - To explore the evolutionary correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
  - Prunes the Halictidae phylogeny to test hypotheses with two independent origins of eusociality.
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** Phylogenetic tree file
  - **`halictidae_pruned_processed_pheno_df.csv`:** Processed phenotype data. 
- **Output Files:**
  - **`halictidae_pruned_dep_x_fitMk_ARD_fitzjohnpi_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)
  
  
### 10) halictidae_pruned_hap_and_eus_depx_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the phylogeny where the Halictidae family is pruned to "force" only two origins of eusociality.. This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depx** in the file name indicates that "sociality" is the dependent variable in the analysis. In this script, haplodiploidy is treated in the strict sense, using only **HD.arrhenotoky**.
- **Format(s):** `.R`  
- **Size:** 8.1 KB  
- **Purpose:** 
  - To explore the correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
  - Prunes the Halictidae phylogeny to test hypotheses with two independent origins of eusociality.
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** Phylogenetic tree file.
  - **`halictidae_pruned_processed_pheno_df.csv`:** Processed phenotype data. 
- **Output Files:**
  - **`halictidae_pruned_dep_x_fitMk_ARD_fitzjohnpi_haparrhenotoky_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)
  
  
### 11) halictidae_pruned_hap_and_eus_depx_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_20241216.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the phylogeny where the Halictidae family is pruned to "force" only two origins of eusociality.. This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depx** in the file name indicates that "sociality" is the dependent variable in the analysis. In this script, haplodiploidy is treated in the strict sense, using **HD.arrhenotoky**, and eusociality is treated in the strict sense, using **Eusocial.strict**.
- **Format(s):** `.R`  
- **Size:** 8.3 KB  
- **Purpose:** 
  - To explore the correlation between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
  - Prunes the Halictidae phylogeny to test hypotheses with two independent origins of eusociality.
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** Phylogenetic tree file
  - **`halictidae_pruned_processed_pheno_df.csv`:** Processed phenotype data. 
- **Output Files:**
  - **`halictidae_pruned_dep_x_fitMk_ARD_fitzjohnpi_haparrhenotoky_euskoos_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)


### 12) halictidae_pruned_hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_20231023.R
- **Description:** An R script designed to test the correlated evolution of Sociality and Ploidy across the phylogeny where the Halictidae family is pruned to "force" only two origins of eusociality.. This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. The term **depxy** in the file name indicates that both "sociality" and "ploidy" are interdependent variables in the analysis.
- **Format(s):** `.R`  
- **Size:** 8.4 KB  
- **Purpose:** 
  - To explore the interdependence between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
  - Prunes the Halictidae phylogeny to test hypotheses with two independent origins of eusociality.
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** Phylogenetic tree file
  - **`halictidae_pruned_processed_pheno_df.csv`:** Processed phenotype data. 
- **Output Files:**
  - **`halictidae_pruned_dep_xy_fitMk_ARD_fitzjohnpi_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform interdependent trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)


### 13) halictidae_pruned_hap_and_eus_depxy_fitMk_ARD_fitzjohnpi_haparrhenotoky_20231023.R
- **Description:** An R script designed to analyze the correlation between the evolution of eusociality and haplodiploidy across a pruned Halictidae phylogeny. This script is optimized for streamlined analysis using the `fitPagel` function and is configured for running on the SOL HPC system at ASU. It includes methods for loading and processing phylogenetic and phenotypic datasets, as well as conducting trait correlation analyses. The term **depxy** in the file name indicates that both "sociality" and "ploidy" are interdependent variables in the analysis. In this script, haplodiploidy is treated in the strict sense, using **HD.arrhenotoky**.
- **Format(s):** `.R`  
- **Size:** 8.2 KB  
- **Purpose:** 
  - To explore the interdependence between eusociality and haplodiploidy in insects.
  - Implements the `fitPagel` function with specific parameters (`method = "fitMk"`, `model = "ARD"`, `pi = "fitzjohn"`).
  - Prunes the Halictidae phylogeny to test hypotheses with two independent origins of eusociality.
- **Input Files:**
  - **`halictidae_pruned_insect_phylogeny_2017_collapsed_calibrated.tre`:** A phylogenetic tree file for Halictidae species.
  - **`halictidae_pruned_processed_pheno_df.csv`:** A processed phenotype dataset containing species-level information.
- **Output Files:**
  - **`halictidae_pruned_dep_xy_fitMk_ARD_fitzjohnpi_haparrhenotoky_calib.RDS`:** Results of the `fitPagel` analysis.
- **Key Libraries Used:**
  - `phytools`
  - `geiger`
- **Workflow:**
  1. Load necessary phylogenetic and phenotypic data.
  2. Configure the analysis environment with specified directories for data and outputs.
  3. Perform interdependent trait correlation analysis using `fitPagel`.
- **References:**
  - [Phytools Pagel's Method Documentation](http://phytools.org/mexico2018/ex/8/Pagel94-method.html)
  - [Blog on `fitPagel` Quirks](http://blog.phytools.org/2016/05/fix-to-peculiar-quirk-of-fitpagel.html)
